server/Dockerfile

```diff
@@ -6,13 +6,14 @@ VOLUME /timely-server-src
 VOLUME /timely
 
 RUN yum -y update && \
-  yum -y install java-1.8.0-openjdk-devel maven openssl apr && \
-  mkdir -p /timely/bin
+  yum -y install java-1.8.0-openjdk-devel maven openssl apr wget && \
+  mkdir -p /timely/bin && \
+  wget -O /usr/lib/jvm/jre/lib/ext/bcprov-jdk15on-155.jar https://www.bouncycastle.org/download/bcprov-jdk15on-155.jar && \
```

Comment:
If there's a base image out there with this already in it, I'd prefer to use that